rdr_service/api/redcap_workbench_audit_api.py

```diff
@@ -57,6 +59,20 @@ def get(self):
         super(RedcapWorkbenchAuditApi, self).get()
         return self.dao.workspace_dao.get_redcap_audit_workspaces(**self.get_filters)
 
+    def _do_insert(self, m):
+        audit_records = super()._do_insert(m)
+        # Generate tasks to build PDR records.
+        if GAE_PROJECT == 'localhost':
+            rebuild_bq_audit(audit_records)
+        else:
+            payload = {'table': 'audit', 'ids': []}
+            for obj in audit_records:
```

Comment:
Do we need to limit the number of ids being rebuilt per task?

Reply:
I have added a 'list_chunks()' for loop to limit batch size.